logrotation/test/k8s/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: s3
+  labels:
+    app: logger-with-rotator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: logger-with-rotator
+  template:
+    metadata:
+      labels:
+        app: logger-with-rotator
+    spec:
+      volumes:
+      - name: logger-local-storage
+        persistentVolumeClaim:
+          claimName: logger-local-pvc
+      # - name: logger-shared-storage
+      #   persistentVolumeClaim:
+      #     claimName: logger-shared-pvc
+      containers:
+      - name: s3  
+        image: logrotate-solution:2.0
+        command: ["/usr/bin/python3"]
+        args: ["/opt/cortx/component/startup/entrypoint_logrotate.py",
+               "--config", "yaml:///opt/cortx/component/services/confstore.yaml"]
+        volumeMounts:
+        # - name: logger-shared-storage
+        #   mountPath: "/var/log/cortx/"
+        - name: logger-local-storage
+          mountPath: "/var/log/cortx/"
+      nodeSelector:
+        disktype: hdd
```

Comment:
why we need this disktype to be mentioned, we need to be agnostic of physical media, from where ever local or shared storage is available for logging

Reply:
sorry, it was just a label I used in my testing. removed it. 👍